lms/tasks/email_digests.py

```diff
@@ -65,7 +66,12 @@ def send_instructor_email_digest_tasks(batch_size):
                 )
 
 
-@app.task
+@app.task(
+    acks_late=True,
+    autoretry_for=(SendDigestsError,),
+    retry_kwargs={"max_retries": 2},
+    retry_backoff=3600,
+)
```

Comment:
`acks_late=True` is [already the default](https://github.com/hypothesis/lms/blob/258614988f659b04db07ff8d01b1d27203bedb7f/lms/tasks/celery.py#L33-L34) in LMS but it's not normally the default for Celery tasks and I wanted to be explicit here in the per-task configuration that we want `acks_late` for this task.